multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/model/SchemaFieldObject.java

```diff
@@ -21,6 +23,39 @@ public class SchemaFieldObject {
 
   private String baseName;
 
+  @Default
+  private BigDecimal maximum = null;
+
+  @Default
+  private BigDecimal minimum = null;
+
+  @Default
+  private Boolean exclusiveMaximum = null;
+
+  @Default
+  private Boolean exclusiveMinimum = null;
+
+  @Default
+  private Boolean uniqueItems = null;
+
+  @Default
+  private Integer minItems = null;
+
+  @Default
+  private Integer maxItems = null;
+
+  @Default
+  private Integer minLength = null;
+
+  @Default
+  private Integer maxLength = null;
+
+  @Default
+  private String pattern = null;
+
+  @Default
+  private String multipleOf = null;
```

Comment:
Thinking about it, it would be preferrable to put all of the information about this fields in a single data structure inside the FieldObject, similar to how we combined the type information in #180.

This way, a method could be used to retrieve a set with the validations that should be ran, which would be significantly more convenient to construct the set you do in the OpenAPI TemplateFactory (L.58-86)

Reply:
Done